frontend/src/js/index.js

```diff
@@ -0,0 +1,3 @@
+import { helloWorld } from "./helloworld";
```

Comment:
Is it possible to import multiple functions within the curly brackets? if so, what would that look like? also: is it best practice to import individual functions, or could you import all functions in a js file, e.g., `import * from "./helloworld"`?